app/views/support/PlanOps.scala

```diff
@@ -50,7 +50,7 @@ object PlanOps {
 
     def isDigitalPack: Boolean = in.product == com.gu.memsub.Product.Digipack
 
-    def isGuardianWeekly: Boolean = in.product == Product.WeeklyZoneA || in.product == Product.WeeklyZoneB // TODO is this right to include both?
+    def isGuardianWeekly: Boolean = in.product.isInstanceOf[Product.Weekly]
```

Comment:
type safety would be lost I think? `==` will give a warning if the types can't ever match but would `isInstanceOf` do that?  another option is something like
```scala
in.product match {
  case Product.Weekly => true
  case _ => false
}```